core/scripts/models/order_book.py

```diff
@@ -93,9 +96,10 @@ def _do_execute(self, msg: Message):
         else:
             self._do_delete(msg)
 
-    def send(self, msg: Message):
+    def send(self, msg: Message, last_sod_offset: int = 0):
```

Comment:
Here the sod_offset could simply be extracted from the message passed as parameter

Reply:
oh right the drowsiness got to me

Reply:
Oh wait no. I did it like this so that the tests would not be affected by it

Reply:
How about adding the default value to the message model instead?

Reply:
Smart man